manifest.yml

```diff
@@ -2,6 +2,7 @@
 applications:
 - name: analytics
   memory: 128M
-  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
-  env:
-    FORCE_HTTPS: true
\ No newline at end of file
+  buildpack: staticfile_buildpack
+- name: analytics-staging
+  memory: 128M
+  buildpack: staticfile_buildpack
```

Comment:
At @dlapiduz' suggestion, I stopped pointing to the Cloud Foundry buildpack and pointed to one maintained internally by the cloud.gov team. I chose to be explicit and reference `staticfile_buildpack`, rather than rely on Cloud Foundry's auto-detection of this buildpack through the presence of `Staticfile`.